Discovery/src/Fortune-Teller-Service/global.json

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
Did you intend to change files from the Discovery sample in this PR?

Reply:
no, will back these changes out